### DISCUSSION TITLE
The Pirate Campaign: Goals and worldbuilding

### DISCUSSION THREAD

**Discussion by Galaucus** [endless-sky/endless-sky#10364] [Ideas]
![image](https://github.com/user-attachments/assets/67d14c29-1849-473c-81bd-40e217670e0c)

### The history of the Republic is one of constant expansion. From its origins as a small alliance of worlds, it has grown to become the dominant power in the known universe. The campaign, and the worldbuilding supporting the campaign, seek to answer the question "what happened to those pushed to the fringes by this expansion?"
Aside from (hopefully) being a thrilling narrative containing fun gameplay and introducing new systems such as smuggling, the pirate campaign, and factions introduced in support of it, will also help explore the black and gray areas of the galactic economy, investigating how the political and economic structures of the Republic and Syndicate have fostered lawlessness and brigandry on the galactic fringes. From slavers and raiders to revolutionaries and rebels, it's a wide variety of outlaws out there, and they all have their role to play in the unfolding drama of the campaign.

# Factions introduced, in summary
### Freebooters

- Operates out of the South. Spawns infrequently in Dirt Belt, Syndicate, and Pirate space.
- Protagonist faction - intended for players wanting a heroic "Robin Hood" or morally grey playstyle
- Hostile towards Lightspeed, attacks merchants on rare occasions
- Hunted by Syndicate, Navy, Deep Security

Based out of the agricultural worlds in the galactic south, the Freebooters are a relatively new as an organization, but has deep roots in the region. Since its earliest days of colonization, people have flocked to the fertile worlds here seeking a new life. Not all would prosper, as frontier living - and keeping up on mortgage and loan payments that covered the cost of migration and settling - would leave some families unable to continue making payments. Their land would be repossessed, sold on to other hopeful colonists, and settlers would be left adrift. If they were lucky, they could find passage to a more industrialized world and find work. If not, they could always sign on with the pirate crews that would visit frontier worlds every so often to sell the goods they'd acquired from raiding.

Some pirate crews became predominantly composed of, or even captained by, former colonists. These pirates, unwilling to raid their former neighbors, would focus their efforts on worlds further north, sometimes even raiding deep into the Syndicate, or less commonly, the industrialized worlds of the galactic rim. Returning to their former homes to unload their plunder in exchange for coin, food, and perhaps news from the old homestead and relatives, tenuous connections were forged with the colonies. In time, these raiders became an in important, though often overlooked part of the colonial economy, acquiring industrialized goods and selling them to colonists at far more affordable prices than they could be bought for otherwise. This has, of course, caused some tension with the greater Republic, and the wealthier colony-worlds of the Rim.

### Lightspeed Logistics

- Headquartered in Syndicate space. Operates throughout the known galaxy, predominantly in Syndicate and Paradise Worlds. Uncommon sight in South and Rim.
- Contains two divisions, Lightspeed Couriers, a civilian shipping firm, and Lightspeed Elite, a grey-market PMC
- Dual-role faction. Non-freebooter players will be able to play a mini-campaign to become a Lightspeed courier / contractor. Within the pirate campaign, they're an antagonist faction.
- Elite division is hostile towards Freebooters, will defend allies from pirates
- Shipping division is hunted by Pirates and Freebooters

A firm with diversified interests, including industrial logistics and commercial shipping, high security armored transport, private security and mercenary contracting, trade-based money laundering, and subscription holostreaming. Lightspeed's illicit activities largely prevent them from seeking a larger share of galactic shipping trade, but they still remain a dominant player within the confines of the Syndicate. Pirates are generally wise enough to leave their armored courier and mercenary divisions alone, though the same can't be said for their general transport fleet.

### Northern Anarchists

- Controls no territory
- Outside of campaign missions, will rarely spawn in scripted (triggered by invisible mission) raids against Barony targets. This is for flavor, though the player can intervene on behalf of one side or the other.
- Story-related faction. If sided with during the campaign endgame, will assume ownership of all Barony worlds.
- Neutral towards all factions, but will protect merchants attacked within their jurisdiction
- Syndicate should be hostile towards them, perhaps Republic, though given how territory is laid out I foresee no interaction

Guerilla remnant of the original northern colonies. Displaced through a series of conflicts throughout the 2960s-2980s, which saw their society - not quite utopian, but certainly idealistic - crushed under the weight of heavily armed warlord forces which would later coalesce into the Baronies. A motley rabble: put ten anarchists in a room and you'll have them proposing twelve competing ideologies for how they should organize themselves, but they remain united in the effort to reclaim their homeland. Increasingly involved in the galactic drug trade as a means to fund their campaign of harassment and eventual reconquest.

### Black Flag Baronies

- Direct government ownership of a new chain of 4-7 stars near the Almaaz cluster, galactic north. Vessels spawn only there.
- Story-related faction, with their fate decided during the campaign's endgame.
- Neutral towards all factions, but will protect merchants attacked within their jurisdiction

In the 2960s, the galaxy was something of a different place. Although the Republic was the only real power in the galaxy, a confederation of anarchist colonies thrived in the north. Though of widely different ideologies, they were generally united in rejection of centralized authority and capitalism, and through economic cooperation and mutual defense. Sufficiently powerful to ward off pirates, projections estimated that within a few decades they could become another galactic power. Republic Intelligence operatives began channeling advanced (for the time) weaponry, funding, and military advisors to eager "freedom fighters" taking a brave stand against the anarchist menace. Though the anarchists were successfully crushed, the now heavily-armed warlords who took their place turned right back around and began raiding the Republic.

Public backlash against this debacle was significant, and of particular outrage to the weapons manufacturers of the Deep, from whom these advanced (though still finicky, as they were early production models) "Thunderclap pulse lasers" were obtained, driving a lasting wedge between the Republic and the Deep. The pulse laser became sufficiently associated in the public consciousness with brutal warlords that it was never put into full production, though rumors persist that secret workshops associated with Republic Intelligence continue to manufacture TCPLs.

Though historically marked by heavy infighting, in recent years a small handful of warlords - dubbing themselves Barons and affecting a generally aristocratic aesthetic in an attempt to legitimate themselves - have managed to subjugate or eliminate their rivals, concentrating power and bringing a measure of stability to the region. No longer needing to raid the Republic to subsist, they've instead begun subjugating the colonies they govern, exporting cheap food and raw materials into Syndicate space for processing. Though nominally free citizens, most inhabitants of Baronic worlds are too impoverished to afford passage anywhere else, and kept subservient by brutally draconian enforcement. Nonetheless, the Barons wish to improve their public image on the galactic stage, become a recognized and legitimate state in the eyes of the Republic, and keep milking their subjugated planets for every credit they're worth.

### Coreside Mineral Concern (CMC)

- Direct ownership of two systems just south of the Syndicate: Gienah, and a new system branching off of Gienah which holds high value asteroids
- Antagonist faction, largely exists to give you targets to shoot at over the course of the campaign
- Neutral towards all factions
- Considers their mineral field system to be private property. Will hail and warn intruders to vacate, then attack.

Criminal cartel that uses slave labor to keep their costs low. Purchases slaves from pirates, works them to death in their mines, and sells the resulting ore to Lightspeed intermediaries who, in turn, sell it on to the Syndicate for processing. Major stakeholders in the company tend to be aristocratic, old-money types, many descended from the old slaver empires from the pre-Republic era. They tend to be a somewhat grumpy bunch, resenting the fact that in the current galactic era they actually have to take an active role in managing their affairs, instead of luxuriating in idle wealth like their forebears.

By no means major, or even minor players on the galactic stage, their involvement in the campaign stems from the grudges and personal ambitions of an influential figure among their ranks.

### Bootleggers

- Extremely minor faction based near the Core
- Direct ownership of one station. Probably best represented using the existing "Independent" government.
- Outlaws, but not pirates
- Features a short mission chain which unlocks outfits for purchase: the Rumjet Afterburner and the Smuggler's Closet.

Near the galactic core lies the Great Molecular Cloud, a vast region of space with an unusually high density of ambient matter. Sufficient quantities of ethanol (also known as drinking alcohol) exist that hydrogen farming operations in the region can be easily retooled to harvest alcohol. A significant industry has been built up on an old refinery station, far from the taxmen and product quality inspectors of the Republic. Here, illegal "Coreshine" is distilled, bottled, and passed on to smugglers for transport into the greater Republic. While generally safe to drink, the purity of any given batch is purely at the discretion of the unregulated distilleries producing them, and it's not uncommon for toxic methanol to be unfiltered from the brew in the name of increased yield and profits...

# Factions, in detail
When creating a character or organization, I like to ask myself three questions: Where is it coming from? What is it in the process of becoming? What drives this change? The answers to these questions drive fleshing out these people and institutions, and govern how they interact within the resulting story.
### Implementation
Ideal implementation includes:
### Implementation
- Two new stations up near Durax. One, _Copperline Station_, is a thriving if troubled smugger's hub and coreshine (space booze) distillery. Its sister station, _Blueside_, has fallen into dereliction and is now used as largely as an isolated drop point for trade, tribute, and black market deals.
- The **rumjet afterburner**, a powerful afterburner model that's rather unsafe to use. It'll inflict hull damage while thrusting.
- The **smuggler's closet**, an outfit that can fully conceal illegal cargo at the cost of significantly reducing overall cargo capacity.
- We might need new sprites for Copperline and Blueside stations.
- Rumjet and closet have sprites done and implemented on the smuggling gameplay test branch.
- Ideally, pirates would only have a modest chance per fleet of being hostile towards bootleggers.

Smugglers, if we implement them as independent spawns at all, would be ignored by most factions. Giving them a low chance to be targeted by lawkeeping forces (Navy, Syndicate, perhaps FW) as well as pirates would be the ideal approach. Perhaps as a name-clone of Merchants with very low spawn rates and different relationships.
### Lore
Bootleggers are not a polity or faction united by ideology, and simply exist as a catch-all term for the smugglers, mechanics, grey market merchants, distillery operators, security personnel, laborers, and operation owners that call Copperline Station home.

### Interactions with other factions
To the extent that they embody any particular political stance (which is slim), they would tend to align with laissez-faire libertarian capitalism. Despite this, they generally hold the Syndicate in low esteem, seeing it as guilty of using "political shenanigans" to maintain its status as a regional monopolist, rather than "honest competition".

Bootleggers, operating on the fringes of the law, must deal with the threat of piracy as a matter of course. There are two dominant approaches: Outrunning them, and paying them off. Conveniently, these work fairly reliably for evading lawkeepers as well.

When dealing with the navy, smugglers prefer to present themselves as honest merchants, and will tend to carry a modest amount of legal commodities in order to mask their true cargo. However, they're not above simply using their agile ships and customized engines and afterburners to simply outrun naval patrols.
### Intentions, goals, and inspirations
The bootleggers provide a minor neutral faction that can be aided or utterly destroyed as the plot demands without changing the galactic status quo. Beyond that, they should ideally provide an interesting target for Republic or Syndicate-aligned players to scan for and intercept, though this may require implementing new gameplay systems and rewards.

The Bootleggers are largely inspired by the historical American phenomenon of the same name, with the stations in particular paying homage to two bluegrass songs: [Good Corn Liquor](https://www.youtube.com/watch?v=vLK55ShcTeQ) and [Blue Side of the Mountain](https://www.youtube.com/watch?v=8Q3Bo_Kz8qQ), while the overall premise (a space station refining alcohol from stellar gases) was inspired by [Molecular Clouds](https://www.youtube.com/watch?v=epGCIKIQMOQ).
### Notes for other writers
Of all the factions, the Bootleggers are the least monolithic. Without any sort of real governing body or even shared interest beyond keeping their space station running and their profits high, they're a motley bunch with varied motivations, allegiances, and foibles. Using bootleggers in your own stories shouldn't require any particular considerations.

**Comment by Saugia** [comment]
With each Pirate faction/section, you can list where they're operating out from (the North, the Core system, the Men area, etc.)

**Comment by Tangle10** [comment]
I acknowledge this is a work in progress, and groups like the CCOR and CMC are likely to get on this as you work on it. On the other hand, I've seen it said that the CCOR is unlikely to be a focus for future campaigns- which to me feels like a missed opportunity. (And naturally the problem with trying to do something myself is a chronic lack of ability to focus on a project...) It strikes me that something involving a stabilized CCOR might be a good years-past epilogue for Free Worlds, something for players who've already done FW dozens of times to stumble upon later- instead of a facet of the Pirate Campaign. (I figure one would effectively be choosing between feasible CCOR governments- Free Worlds puppet, pirate warlord rule, Independent like Thule, something radical...?) 

I imagine a couple factions exist for the thriving anarchist movement- Thule and Freedom are both stated to effectively be under anarchist governing schemes, and iirc both have around a billion(!) people (though Freedom's description makes it sound more like a thousand). It makes sense if they'd be side players to the pirate campaign, but it's definitely something useful to keep in mind if there's to be a full-scale campaign.

**Reply by Saugia** [reply]
"Factions" are planned to exist, but not globally on the map. Similar to various other missions, any faction representation would be within the missions/side-campaign missions themselves rather than on the map directly.

**Reply by Tangle10** [reply]
> "Factions" are planned to exist, but not globally on the map. Similar to various other missions, any faction representation would be within the missions/side-campaign missions themselves rather than on the map directly.

Well, naturally- but especially if there's going to be changes to the galactic status quo, it may well be worth it to have more governments on the map....

**Reply by Saugia** [reply]
We had discussions about that with Lake and occasionally on stream during campfire chats, and ultimately believe that the map is supposed to be sectioned off to generic governments and avoid using more than what's necessary; most if not all of these can be described as Pirates, and Pirates are a generalized thing in human space.

Specific governments are better of within story missions rather than splitting the map up as well as the representation. Technically there could be 7+ factions of Pirates, and the second we make one of them represent a chunk of the map, a lot of baselines could be established that might not be true.

**Reply by Saugia** [reply]
It wouldn't really change a whole lot in the status quo per-say; government wise the map will play out as it always has (and pirate campaign is still based on the events of the main campaign). An area won't strictly change allegiance to another government, for example, and if it were, it would be more something like the "influence of pirates in the area is strongly this-faction based", not really any overhead ownership of an area as concrete as something like the Republic.

**Reply by Tangle10** [reply]
> It wouldn't really change a whole lot in the status quo per-say; government wise the map will play out as it always has (and pirate campaign is still based on the events of the main campaign). An area won't strictly change allegiance to another government, for example, and if it were, it would be more something like the "influence of pirates in the area is strongly this-faction based", not really any overhead ownership of an area as concrete as something like the Republic.

Right, which completely contravenes the idea of something like "nice pirates" outside playing as a pirate. Unless you're willing to make various systems Independent, but it seems that the idea is that all statuses will be equivalently quo at the end of each campaign.

**Reply by Ebreus** [reply]
CCOR Space is pretty heavily populated and up to relatively recently was a unified interstellar nation in ideological opposition to the Republic. CCOR may've fallen but not everyone accepts that (guerilla forces are fighting to restore it) and few accept the current order (petty and corrupt governments ruling over fragments of CCOR).

Last I heard it's planned that one of the Pirate Campaign endings will see the reformation of CCOR. Though considering Pirate & FW campaigns take place simultaneously and _both still happen_ regardless of if you're a Pirate or FWer it would be neat to see reformation be the default assumption with FW <first><last> having a side role to play in it happening.

**Comment by Galaucus** [comment]
Summaries of all factions are written and in a place I feel good presenting, I consider the broad strokes of this post now open to discussion. Next task is to write up specific implementation goals for each group, including:
- Bare minimum implementation requirements
- Ideal "Lake's vision" implementation requirements
- Intentions, goals, and inspirations
- Detailed backstories and lore implications
- Potential points of controversy
- Notes for other writers on making content interacting with these groups

I'll also aim to put together a graphical timeline showing major events in galactic history, how they shaped the status of the outlaw fringe, and where current campaign-related factions fit into things.

**Reply by Tangle10** [reply]
I do note that you haven't put the CCOR in that list of faction summaries- is that because it's already in, or because it'll be at most a sideline/tangent to the Pirate campaign, or...?

**Reply by Galaucus** [reply]
> I do note that you haven't put the CCOR in that list of faction summaries- is that because it's already in, or because it'll be at most a sideline/tangent to the Pirate campaign, or...?

Already implemented and thoroughly discussed, keeping things focused on the not-yet-in here.

**Reply by Tangle10** [reply]
> > I do note that you haven't put the CCOR in that list of faction summaries- is that because it's already in, or because it'll be at most a sideline/tangent to the Pirate campaign, or...?
> 
> Already implemented and thoroughly discussed, keeping things focused on the not-yet-in here.

Makes sense!

**Comment by Galaucus** [comment]
Bootleggers section is largely complete. May revisit it later to adjust the fine details, but I consider that subsection ready for review.
